control-plane/api/v1alpha1/routeauthfilter_types.go

```diff
@@ -26,8 +26,8 @@ type RouteAuthFilter struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   RouteAuthFilterSpec `json:"spec,omitempty"`
-	Status `json:"status,omitempty"`
+	Spec   RouteAuthFilterSpec   `json:"spec,omitempty"`
+	Status RouteAuthFilterStatus `json:"status,omitempty"`
```

Comment:
Is there a reason to have a specific Status type here? The other types in consul all seem to use the common corev1.Status struct.